### DISCUSSION TITLE
How can i place popper in left or right side using Material Design

### DISCUSSION THREAD

**Discussion by Mateus-Felipe-Ribeiro** [hakimio/ngx-ui-tour#107] [Q&A] [answered]
Hello, i using Material Design for my app, but for now it is only possible to place above or below.

Is there any possible way to put left or right?

**Comment by hakimio** [comment]
You can specify `xPosition` and `yPosition` described in [the `mat-menu` docs](https://material.angular.io/components/menu/api#MenuPositionX).
```ts
import {IStepOption} from 'ngx-ui-tour-md-menu';

const steps: IStepOption[] = [{
    anchorId: 'start.tour',
    content: 'Welcome to the Ngx-UI-Tour tour!',
    title: 'Welcome',
    placement: {
       xPosition: 'before',
       yPosition: 'above'
    }
}];
```

However, it will only work if there is enough space to show the menu on the specified position.

**Comment by hakimio** [comment, marked as answer]
A bit late, but... now there is also `placement.horizontal` config.

**Reply by Mateus-Felipe-Ribeiro** [reply]
Perfect, thanks a lot.
